packages/perseus-core/src/parse-perseus-json/perseus-parsers/perseus-answer-area.ts

```diff
@@ -0,0 +1,37 @@
+import {object, pipeParsers} from "../general-purpose-parsers";
+import {convert} from "../general-purpose-parsers/convert";
+import {defaulted} from "../general-purpose-parsers/defaulted";
+
+import type {PerseusAnswerArea} from "@khanacademy/perseus-core";
+
+export const parsePerseusAnswerArea = pipeParsers(
+    defaulted(object({}), () => ({})),
+).then(convert(toAnswerArea)).parser;
+
+// Some answerAreas have extra, bogus fields, like:
+//
+//   "answerArea": {
+//     "type": "multiple",
+//     "options": {},
+//     "version": null,
+//     "static": false,
+//     "graded": false,
+//     "alignment": "",
```

Comment:
That's interesting, probably a bug with serialization. Some of our serialization is based on a blocklist of properties rather than an allowlist, which occasionally introduces some accidental properties.